### ISSUE TITLE
Problems using -a option for archives 

### ISSUE THREAD

**Issue by rflgrt** [deadc0de6/gocatcli#2]
When using the -a option and afterwards looking at the catalog using gocatcli nav, the archive has the correct number of entries, but the files all have the same name as the archive. 

The files in the archive can not be found using gocatcli fzfind.

Tested on Ubuntu 23.10 using v1.0.1 using *.zip and *.tar.gz archives packed using Engrampa.

Best wishes

**Comment by userwiths** [comment]
I'd love to submit a fix, but in case the maintainer has more free time than me.
I believe the issue to be related to the [following logic](https://github.com/deadc0de6/gocatcli/blob/1bdc66e98f25d3e7be57c1bd8c474fc248e3af46/internal/stringer/out_native.go#L54) I have changed locally the condition to 
```
if sto != nil && n.GetType() == "archive" {
       name = fmt.Sprintf("%-60s", filepath.Join(sto.GetName(), n.GetPath()))
}
```
And believe it is behaving correctly or at least better. Will try to find time to submit a PR but don't promise anything.

\* Maybe this behavior should be changed in the other `stringers` too :thinking: .

**Comment by deadc0de6** [comment]
Thanks for submitting this.
It was indeed a bug in the way I was handling the archived files. It affected `nav` and `fzfind` but also indirectly `mount` and `create`. Everything should be fixed in version [v1.0.2](https://github.com/deadc0de6/gocatcli/releases/tag/v1.0.2).

@userwiths thanks also for your help. The bug was actually located in the way archived files are returning their paths (the `GetPath` method) as they contain their parent (archive file) as path and not their name. It's should be all fixed.
